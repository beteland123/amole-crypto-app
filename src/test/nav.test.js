import { render } from '@testing-library/react';
import {
  MemoryRouter, Routes, Route,
} from 'react-router-dom';
import NAVBAR from '../components/navbar';

describe('test navabar', () => {
  test('it should match the snapshoot', () => {
    const { container } = render(
      <MemoryRouter initialEntries={['/navbar']}>
        <Routes>
          <Route path="/navbar" element={<NAVBAR />} />
        </Routes>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
