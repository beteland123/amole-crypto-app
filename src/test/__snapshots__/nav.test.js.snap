// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test navabar it should match the snapshoot 1`] = `
<div>
  <nav
    class="navbar lato"
  >
    <li>
      <a
        class=""
        href="/"
      >
        <img
          alt="back"
          class="App-logo"
          src="back.png"
        />
      </a>
      <a
        class=""
        href="/id"
      />
    </li>
    <li>
      <h1
        class="title"
      >
        COIN STATS
      </h1>
    </li>
    <li
      class="icon"
    >
      <img
        alt="micro"
        class="App-logo"
        src="microphone.png"
      />
      <img
        alt="gear"
        class="App-logo"
        src="gear.png"
      />
    </li>
  </nav>
</div>
`;
